tests/unit/application/engine/test_autopilot_act_planning_unbound.py

```
@@ -0,0 +1,67 @@
+"""回归测试：_handle_act_planning 中 rec_chapters_per_act 的 UnboundLocalError。
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)